website/resources/components/observability.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Arcus Observability
-description: Observability with Microsoft Azure in a breeze.
+description: Arcus Observability allows you to work easily with Azure Application Insights telemetry. Instead of managing logging, correlation, tracking, Arcus Observability allows you to work with the common ILogger infrastructure and still be able to write multi-dimensional telemetry data. 
```

Comment:
```suggestion
description: Arcus Observability allows you to work easily with Azure Application Insights telemetry.  Arcus Observability allows you to work with the common ILogger infrastructure to track dependencies, log custom metrics and log multi-dimensional telemetry data. 
```

Reply:
This does not feel as an improvement but as duplication. Could you maybe point the problem that you're having with the original?

Reply:
I think it is more clear.   It says what you can do with Observability instead of saying what you don't need to do.

Reply:
Ok, but I'll remove the duplication. So, something like this:

`Arcus Observability allows you to work easily with Azure Application Insights telemetry that makes use of the common ILogger infrastructure to track dependencies, log custom metrics and log multi-dimensional telemetry data.`

Reply:
Ah, with the duplication you referred to the fact that both sentences begin with 'Arcus Observability allows ... '

Reply:
Done in https://github.com/arcus-azure/arcus/pull/278/commits/da644ae680dd38d9d6fb40c067c7d993b7062166